test/helpers/typeExt.js

```diff
@@ -1,39 +0,0 @@
-
```

Comment:
This file isn't used anywhere. `import {zip} from './func';` throws an error when running `truffle test`